tests/subsys/pm/power_mgmt/src/main.c

```diff
@@ -304,7 +319,8 @@ void test_power_state_trans(void)
 	k_sleep(SLEEP_TIMEOUT);
 	zassert_true(leave_idle, NULL);
 
-	pm_device_runtime_enable(device_dummy);
+	ret = pm_device_runtime_enable(device_dummy);
+	zassert_true(ret == 0, "Failed to enable device runtime PM");
```

Comment:
```suggestion
	zassert_equal(ret, 0, "Failed to enable device runtime PM");
```

Reply:
it is consistent with the previous check